app/notifications/developers/product_feature_notification.rb

```diff
@@ -0,0 +1,28 @@
+module Developers
+  class ProductFeatureNotification < ApplicationNotification
+    deliver_by :database, if: :deliver_notification?
+    deliver_by :email, mailer: "DeveloperMailer", method: :product_feature_update, if: :deliver_notification?
+
+    param :developer
+
+    def developer
+      params[:developer]
+    end
+
+    def title
+      t("notifications.developers.product_feature_notification.title")
+    end
+
+    def email_subject
+      t("notifications.developers.product_feature_notification.email_subject")
+    end
+
+    def url
+      edit_developer_url(developer, anchor: "notifications")
```

Comment:
Let's add an anchor `id` tag to the developer form so we can link directly to the new field.